src/nplinker/strain_collection.py

```diff
@@ -210,3 +210,9 @@ def __str__(self):
 
         return f'StrainCollection(n={len(self)}) [' + ','.join(
             s.id for s in self._strains) + ']'
+    
+    def __eq__(self, other):
+        result = self._strains == other._strains
```

Comment:
the `self._strains` has the type `list[Strain]`. so the order of Strain objects in the list also matters for the comparison. Do we care about the order? Does it make more sense to change the type to `set[Strain]`?

Reply:
I guess we shouldn't care about the order - so lets create an issue to see how it works when changing the type to `set[Strain]`, but let's keep this separate.